tests/cpp/test_multiple_registered_engines.cpp

```diff
@@ -1,68 +0,0 @@
-#include <string>
-#include "gtest/gtest.h"
-#include "tests/util/util.h"
-#include "torch/script.h"
-#include "torch_tensorrt/torch_tensorrt.h"
-
-#ifndef DISABLE_TEST_IN_CI
```

Comment:
Also another test to keep and check if it works with cos similarity

Reply:
I moved this to python  as well to compare with nn.Module. Check out tests/models/multiple_registered_engines.py

Reply:
Again, need to verify the c++ api

Reply:
added it back